src/main/plugins/org.dita.xhtml/xsl/xslhtml/rel-links.xsl

```diff
@@ -600,6 +601,12 @@ Each child is indented, the linktext is bold, and the shortdesc appears in norma
   </xsl:template>
 
   <!--linklists-->
+  <xsl:template match="*[contains(@class, ' topic/linklist ')]/@xml:lang" priority="100">
```

Comment:
This should serve the same purpose as the change in the HTML file. (I didn't intend to check in the priority and should fix that.) However: our root `dita2xhtml.xsl` file overrides processing for `@xml:lang`, and regardless of what priority I set here, any stylesheet call entering with that file gets the processing from that root file. That's why all of the XHTML integration test results had to be updated. 

I originally considered moving that processing from the shell into `dita2htmlImpls.xsl`, which more or less matches the HTML5 organization, but there is a very slight chance of introducing an incompatibility with that.

Any shell using the base XHTML rules, without that `dita2xhtml.xsl` shell, will be able to use this override here; given the status of XHTML output, and that duplicating the lang attribute isn't actually a problem, I decided to just fix the tests.